lighthouse-core/test/audits/bootup-time-test.js

```diff
@@ -7,59 +7,63 @@
 
 /* eslint-env mocha */
 const BootupTime = require('../../audits/bootup-time.js');
-const fs = require('fs');
+const DevtoolsTimelineGather = require('../../gather/computed/devtools-timeline-model');
 const assert = require('assert');
+const {groups} = require('../../lib/task-groups');
+
+const devtoolsTimelineGather = new DevtoolsTimelineGather({});
```

Comment:
how about we use a real `Artifacts` object to test the real flow 

i.e. similar to 
https://github.com/GoogleChrome/lighthouse/blob/ac2bc69936b2e7160f9a62b08195829762070e14/lighthouse-core/test/audits/consistently-interactive-test.js#L32-L39